### ISSUE TITLE
THREE.BufferGeometry.addAttribute: Use .setIndex() for index attribute.

### ISSUE THREAD

**Issue by arjanfrans** [Jam3/three-bmfont-text#2]
I use three.js version 0.73 and it gives the following warning:

```
THREE.BufferGeometry.addAttribute: Use .setIndex() for index attribute.
```

**Comment by mattdesl** [comment]
Thanks; should probably figure out how to migrate this module to newest ThreeJS; maybe some compatibility layer, or maybe just bump major version and point users on r69 to old version.

**Comment by mattdesl** [comment]
This should be fixed in `2.0.0` but be sure to check the [change log](https://github.com/Jam3/three-bmfont-text#change-log) as there are some breaking changes.
